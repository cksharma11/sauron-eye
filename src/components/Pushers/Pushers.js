import React from "react";
import PusherInfo from "../PusherInfo/PusherInfo";

const Pushers = ({ dashboard }) => {
  return dashboard.map(pusher => {
    return <PusherInfo pusher={pusher} key={pusher}/>;
  });
};

export default Pushers;
