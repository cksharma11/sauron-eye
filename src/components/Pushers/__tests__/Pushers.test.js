import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Pushers from "../Pushers";

configure({ adapter: new Adapter() });

describe("Pushers", () => {
  it("should render properly", () => {
    const dashboard = [
      {
        name: "somename",
        passed: 10,
        failed: 5
      },
      {
        name: "somename",
        passed: 10,
        failed: 5
      }
    ];
    const wrapper = shallow(<Pushers dashboard={dashboard} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
