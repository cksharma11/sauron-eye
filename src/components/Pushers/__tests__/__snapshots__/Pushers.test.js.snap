// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pushers should render properly 1`] = `
"<PusherInfo pusher={{...}} number={1} />


<PusherInfo pusher={{...}} number={2} />"
`;
